.github/workflows/on_release_publish.yml

```diff
@@ -161,42 +161,63 @@ jobs:
     env:
       DOCKERHUB_ORG: gospatial
       DOCKERHUB_REPO: tegola
+      BUILD_ARGS: |
+        BUILDPKG=${BUILD_PKG}
+        VER=${VERSION}
+        BRANCH=${GIT_BRANCH}
+        REVISION=${GIT_REVISION}
 
     steps:
     - name: Check out code
       uses: actions/checkout@v3
 
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
     - name: Setup env
       uses: ./.github/actions/tegola-setup-env
       with:
         ui: true
 
-    - name: Build and tag Docker container
+    - name: Build and export to Docker
+      uses: docker/build-push-action@v4
+      with:
+        context: .
+        load: true
+        platforms: linux/amd64
```

Comment:
did you intentionally leave out `linux/arm64` here? If so, why?

Reply:
Basically, when you build an image with `buildx`, by default, it will not be available for usage, eg, starting actual container with it, so here I'm passing option `load: true` so that it will output image into the docker, but it [can only output single-platform result image](https://docs.docker.com/engine/reference/commandline/buildx_build/#output:~:text=The%20docker%20export%20type%20writes%20the%20single%2Dplatform%20result%20image%20as%20a%20Docker%20image%20specification%20tarball%20on%20the%20client.%20Tarballs%20created%20by%20this%20exporter%20are%20also%20OCI%20compatible.), so for the purpose of testing if image is build properly, I have to do one build here, and later make another builds that will actually be pushed to Docker Hub (At least that is what official [docs](https://docs.docker.com/build/ci/github-actions/test-before-push/) are suggesting)

Reply:
I see. Please add some code comments here so the future us understands what's going on.

Reply:
Done. Hope comment is clear enough.

Reply:
Looks good to me.